UrbanMobility/StopTime/schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/UrbanMobility/StopTime/schema.json",
+  "title": "NGSI GTFS Schema - StopTime",
+  "description": "GTFS Stop Time",
+  "type": "object",
+  "allOf": [
+    { "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/GSMA-Commons" },
+    {
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "gtfs:StopTime"
+          ],
+          "description": "NGSI Entity type"
+        },
+        "hasStop": {
+          "type": "string",
+          "format": "uri"
+        },
+        "hasTrip": {
+          "type": "string",
+          "format": "uri"
+        },
+        "arrivalTime": {
+          "type": "string"
```

Comment:
pattern: "^([0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]$"

Reply:
goos idea. I will check the pattern so that it matches Google format